Library/Feature.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public enum Feature: String {
+  case nativeCheckout = "ios_native_checkout"
```

Comment:
Not sure why this file appears new, I just moved it.

Renamed `checkout` ->  `nativeCheckout` here to better match the feature flag name and for consistency with the flag just added.